apps/console/src/features/applications/pages/applications.tsx

```diff
@@ -276,6 +285,75 @@ const ApplicationsPage: FunctionComponent<ApplicationsPageInterface> = (
         return <RemoteFetchStatus data-testid={ "remote-fetch" } />;
     };
 
+    /**
+     * Renders the URL for the tenanted my account login.
+     *
+     * @return {React.ReactElement}
+     */
+    const renderTenantedMyAccountLink = (): ReactElement => {
+        if (AppConstants.getTenant() === AppConstants.getSuperTenant()) {
+            return null;
+        }
+
+        return (
+            <EmphasizedSegment
+                className="mt-0 mb-5"
+                data-testid={ `${ testId }-consumer-account-link` }
```

Comment:
Use the component id instead.
```suggestion
                data-testid={ `${ testId }-consumer-account-link` }
```

Reply:
Fixed with c2cc6d9fb5a4ad0937b45edc1fe14ea5f66fb6d3